dev/TabView/TabView.cpp

```diff
@@ -325,13 +326,46 @@ void TabView::OnListViewSelectionChanged(const winrt::IInspectable& sender, cons
 
     UpdateTabContent();
 
-    m_selectionChangedEventSource(sender, args);
+    m_selectionChangedEventSource(*this, args);
 }
 
+winrt::TabViewItem TabView::FindTabViewItemFromDragItem(const winrt::IInspectable& item)
+{
+    auto tab = ContainerFromItem(item).try_as<winrt::TabViewItem>();
+
+    if (!tab)
```

Comment:
What are the cases where ContainerFromItem will fail to give the required container?

Reply:
Unfortunately, this happens during drag and drop. There's a period of time after the drop where ContainerFromItem will NOT return the item even if it is a TabViewItem. Once we switch to Repeater hopefully we can get rid of a lot of this ridiculousness.